auth/api/grpc/server.go

```diff
@@ -39,7 +45,7 @@ type grpcServer struct {
 }
 
 // NewServer returns new AuthServiceServer instance.
-func NewServer(svc auth.Service) magistrala.AuthServiceServer {
+func NewServer(svc auth.Service) AuthServiceServer {
```

Comment:
@felixgateru Can we have seperate NewServer for Auth and Policy ?

Reply:
Yes, I will seperate them into two servers.